File: templates/BenchmarkDotNet.BenchmarkPorjectTemplate.CSharp/content/_BenchmarkName_.cs
using System;
using BenchmarkDotNet;
using BenchmarkDotNet.Attributes;

namespace _BenchmarkProjectName_
{
#if(config)
    [Config(typeof(BenchmarkConfig))]
#endif
    public class $(BenchmarkName)
    {

        [Benchmark]
        public void YourBenchmark()
        {
            // Implement your benchmark here

            
        }
    }
}

File: templates/BenchmarkDotNet.BenchmarkProjectTemplate.VB/content/_BenchmarkName_.vb
Imports System
Imports BenchmarkDotNet
Imports BenchmarkDotNet.Attributes

Namespace _BenchmarkProjectName_

#If config Then
    <Config(GetType(BenchmarkConfig))>
#End If
    Public Class $(BenchmarkName)
        <Benchmark>
        Public Sub YourBenchmark()
        
        ' Implement your benchmark here

        End Sub
    End Class
End Namespace